newsfragments/2796.md

```diff
@@ -0,0 +1 @@
+Mixing cfgs and pyo3 attributes on struct fields will now work
```

Comment:
Needs to be 2796.changed.md (category is important so that towncrier knows where this goes on the changelog).